src/js/resize-manager.js

```diff
@@ -80,7 +80,9 @@ class ResizeManager extends Component {
 
   createEl() {
     return super.createEl('iframe', {
-      className: 'vjs-resize-manager'
+      'className': 'vjs-resize-manager',
+      'tabIndex': -1,
+      'aria-hidden': true
```

Comment:
ARIA attributes should go in the third parameter to `createEl`, and take a string value not a boolean. E.g. (https://github.com/videojs/video.js/blob/70a71ae81e5026faf38fc0d675521d66847d8411/src/js/control-bar/progress-control/play-progress-bar.js#L24):

```js
    return super.createEl('div', {
      className: 'vjs-play-progress vjs-slider-bar'
    }, {
      'aria-hidden': 'true'
    });
```